frontend/lib/models/transaction.dart

```diff
@@ -1,145 +1,48 @@
+import 'package:economicalc_client/models/receipt.dart';
+import 'package:economicalc_client/models/bank_transaction.dart';
+import 'package:intl/intl.dart';
+
 class Transaction {
-  late String id;
-  late String accountId;
-  late Amount amount;
-  late Descriptions descriptions;
-  late Dates dates;
-  late Types types;
-  late String status;
-  late String providerMutability;
+  int? id;
+  DateTime date;
+  double? totalAmount;
+  String? store;
+  int? bankTransactionID;
```

Comment:
Why not attach an object of class BankTransaction here?

Reply:
There is no need to attach the whole object, we can just fetch it from the db when needed. We also don't want to store duplicates of the objects in the db, it would cause synchronization issues.